math/binarymath/checkisnumberpoweroftwo.go

```diff
@@ -0,0 +1,37 @@
+// checkisnumberpoweroftwo.go
+// description: Is the number a power of two
+// details:
+// Checks if a number is a power of two- [Power of two](https://en.wikipedia.org/wiki/Power_of_two)
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see checkisnumberpoweroftwo_test.go
+
+package binarymath
```

Comment:
This is me nitpicking but could you change the name of this package?

`math/binarymath` sounds odd - it would be much better to have `math/binary`. Don't you think?

Reply:
good idea